R/ae_attendances_admissions_age_sex.R

```diff
@@ -29,6 +29,8 @@ ae_attendances_admissions_age_sex <- function(start_date, end_date, data, plot_c
     dplyr::filter(spell_start <= end_date & spell_end >= start_date) %>%
     dplyr::select(spell_number, spell_start, spell_end, gender, age_band_start, spell_class_col)
 
+  dt <- dt %>%
+    dplyr::mutate(age_band_start = factor(age_band_start, levels = gtools::mixedsort(unique(dt$age_band_start)))) # necessary for correct age band ordering on x axis
```

Comment:
@Wartynas I think this should now be ... `levels = tools::mixedsort(unique(levels(dt$age_band_start))` - see my comment on the main thread of the PR

Reply:
Ah no hold on - I've just realised at this point age_band_start is not a factor... bear with me...